### ISSUE TITLE
Vanity address creation?

### ISSUE THREAD

**Issue by xanoni** [PurpleI2P/i2pd#1697]
Does i2pd have built-in functionality for vanity address generation (`foobarxyz[...].b32.i2p`)?

I noticed that i2p (Java) has this feature (`server.create.vanity`), however I also [noticed some code in this `i2pd` repo here](https://github.com/PurpleI2P/i2pd/commit/04297eda8084d57f539cef70869eeab009a73c84).

Is this functional? [I couldn't see it mentioned in your docs](https://i2pd.readthedocs.io/en/latest/) and [the diff](https://github.com/PurpleI2P/i2pd/commit/04297eda8084d57f539cef70869eeab009a73c84) is a bit tough to read just with the tools that GitHub offers given all the formatting changes ...

If not built-in, is there a good external tool (like [mkp224o](https://github.com/cathugger/mkp224o) for Tor)?

**Comment by r4sas** [comment]
https://github.com/PurpleI2P/i2pd-tools/blob/master/vanitygen.cpp

**Comment by xanoni** [comment]
Makes sense. I noticed `FUCKING PLAZ!` mentioned in a comment in the code and assumed it was some Russian curse word 🤣

**Comment by MoneroArbo** [comment]
Why was the vanity address generator removed recently?

https://github.com/PurpleI2P/i2pd-tools/commit/74173a26b117dce6ac56caebcf106d72d299180b
